Purchases/RCHTTPClient.m

```diff
@@ -8,75 +8,106 @@
 
 #import "RCHTTPClient.h"
 #import "RCUtils.h"
-#import "RCPurchases.h"
```

Comment:
the diff in this file might be easier to visualize in split view because there's a lot of stuff moved around through refactoring